kmir/Makefile

```diff
@@ -116,6 +116,7 @@ check-black: poetry-install
 # Kompilation
 
 kbuild-%:
-	$(POETRY_RUN) kbuild kompile $*
+	$(POETRY_RUN) kbuild kompile $* > $*.def; cat $*.def
```

Comment:
I'm not sure if this is the best way to do this. I was having trouble with environment variables. `cat $*.def` is only there so when building it still prints the location of the definition, but this isn't strictly necessary.